### ISSUE TITLE
Initialising Model

### ISSUE THREAD

**Issue by transfluxus** [MediaUncovered/explorer-server#4]
hi I  have problems starting the app and not sure if its a problem in the docker config
- I cloned the repo
- go into the project folder
- build the docker
- ran `docker run -v app:/flask-explorer/app -p 5000:5000 -t media-uncovered/explorer-server:dev`
and different variations all resulting in 
`FileNotFoundError: [Errno 2] No such file or directory: '/flask-explorer/models/model'`

what is the project name? explorer-server, explorer-flask or flask-explorer? :)

**Comment by transfluxus** [comment]
ok, actually I like the idea of running it localy in a python environment much more.
So
```
  File "server.py", line 1, in <module>
    from app import create_app
  File "/projects/EXPLORE/explorer-server/app/__init__.py", line 5, in <module>
    from newsAnalysis.Model import Model
ModuleNotFoundError: No module named 'newsAnalysis'
```
I cloned newsAnalysis into a 2nd folder and installed its requirements in the same env. 
What's the smoothest way to make the server find newsAnalysis?

**Comment by transfluxus** [comment]
BREAKING:
I got it running by using pycharm.
However just running it will crash for not finding a model.
I copied the MoscowTimes_1000.pkl into the models folder and changed the Model.init to eat a `name` and `model_path` file.
No way to load the model with or without (.pkl) filetype ending even after removing
the magic filetype string in the load function:
`input_file = open(model_path + '.pkl', 'rb')`.
ending with an error in gensim.
```
  File "/media/ra/hd2/projects/EXPLORE/explorer-server/venv/lib/python3.6/site-packages/gensim/utils.py", line 426, in load
    obj._load_specials(fname, mmap, compress, subname)
AttributeError: 'Model' object has no attribute '_load_specials'
```
Is that model compatible with the gensim?

don't use magic filetype strings :)
